pkg/wfc3/calwf3/wf3cte/wf3cte.c

```diff
@@ -996,7 +996,7 @@ int find_dadj(int i ,int j, double obsloc[][RAZ_ROWS], double rszloc[][RAZ_ROWS]
     dval9 = 0.;
 
     /*COMPARE THE SURROUNDING PIXELS*/
-    if (i==1 &&  RAZ_ROWS-1>=j  && j>0 ) {
```

Comment:
Just to answer the specific question here:  Yes, when j equals RAZ_ROWS - 1, one element is read from both obsloc and rszloc that is out of bounds of these arrays.  There are two other points (i.e. uses of j+1) where the wrong array element is read, but those are at least within the bounds of the arrays.